LibraryManagement/Class/Admin.vb

```diff
@@ -1,4 +1,4 @@
 ﻿Public Class Admin
-    Public UID As String
+    Public Uid As String
```

Comment:
So here it is "Uid" and everywhere in the program its like "uid"

Reply:
VB is not case sensitive.

Reply:
we'll fix it in the final clean up

Reply:
ok